examples/nametoken/tests/tests.rs

```diff
@@ -0,0 +1,712 @@
+//! Tests for the `nametoken` contract.
+use concordium_cis2::*;
+use concordium_smart_contract_testing::*;
+use nametoken::*;
+
+/// The tests accounts.
+const ALICE: AccountAddress = AccountAddress([0; 32]);
+const ALICE_ADDR: Address = Address::Account(ALICE);
+const BOB: AccountAddress = AccountAddress([1; 32]);
+const BOB_ADDR: Address = Address::Account(BOB);
+const CHARLIE: AccountAddress = AccountAddress([2; 32]);
+
+/// Token IDs.
+const NAME_0: &str = "MyEvenCoolerName";
+const NAME_0_TOKEN_ID: ContractTokenId = TokenIdFixed([
+    24, 188, 23, 153, 239, 160, 1, 1, 235, 169, 225, 59, 20, 47, 103, 115, 3, 188, 0, 164, 87, 161,
+    21, 92, 67, 206, 126, 235, 0, 101, 54, 231,
+]);
+const NAME_1: &str = "MyCoolName";
+const NAME_1_TOKEN_ID: ContractTokenId = TokenIdFixed([
+    183, 115, 55, 205, 199, 72, 252, 206, 49, 209, 119, 201, 194, 140, 103, 91, 216, 188, 175, 84,
+    169, 226, 17, 22, 227, 62, 16, 117, 35, 207, 236, 137,
+]);
```

Comment:
Maybe we could implement `FromStr` for TokenIdFixed (we already have a hex-based Display).

That'll simplify tests.

Reply:
`FromStr::from_str` is not a `const` function, so that wouldn't help much in this case.
But it seems like I could implement a `const` method that would allow us to write `TokenIdFixed::convert_to_hex_and_parse(NAME_1)` by using https://docs.rs/const-hex/latest/const_hex/fn.const_encode.html and https://docs.rs/const-decoder/latest/const_decoder/.

But this might be too heavy a solution for now.
What do you think?

Reply:
Yeah, not for now.